Dockerfile

```diff
@@ -27,13 +27,18 @@ RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && \
 	php5-cli \
 	php5-pgsql \
 	# Ruby
-	ruby-pg \
+	rbenv \
 	&& rm -rf /var/lib/apt/lists/*
 
 RUN pip install psycopg2
 
 RUN npm install pg@5.0.0
 
+RUN git clone https://github.com/rbenv/rbenv.git ~/.rbenv && \
```

Comment:
this should `--shallow --branch v1.1.0`. I'm also not a fan of rbenv, but I don't feel strongly.